js/app4.js

```diff
@@ -0,0 +1,201 @@
+var paris = new Location('Paris', 20, 38, 2.3);
+var lima = new Location('Lima', 2, 16, 4.6);
+
+
+// Invoking Functions
+
+seattle.renderData();
+tokyo.renderData();
+dubai.renderData();
+paris.renderData();
+lima.renderData();
+
+
+// Console Log
+
+console.log(seattle, tokyo, dubai, paris, lima);
+
+
+// ROW TWO: Seattle
+// Method: give an array, pass into the function this.cookiesPerHour.length[i], pass data through the method
+
+var tableRow = document.createElement('tr');
+var citySeattle = document.createElement('th');
+citySeattle.textContent = seattle.name;
+tableRow.appendChild(citySeattle);
+for (var i = 0; i < seattle.cookiesPerHour.length; i++){
+  var tableData = document.createElement('td');
+  tableData.textContent = seattle.cookiesPerHour[i];
+  tableRow.appendChild(tableData);
+}  
+parentElement.appendChild(tableRow);
+
+
+// ROW THREE: Tokyo
+
+var tableRow = document.createElement('tr');
+var cityTokyo = document.createElement('th');
+cityTokyo.textContent = tokyo.name;
+tableRow.appendChild(cityTokyo);
+for (var i = 0; i < tokyo.cookiesPerHour.length; i++){
+  var tableData = document.createElement('td');
+  tableData.textContent = tokyo.cookiesPerHour[i];
+  tableRow.appendChild(tableData);
+}  
+parentElement.appendChild(tableRow);
+
+
+// ROW FOUR: Dubai
+
+var tableRow = document.createElement('tr');
+var cityDubai = document.createElement('th');
+cityDubai.textContent = dubai.name;
+tableRow.appendChild(cityDubai);
+for (var i = 0; i < dubai.cookiesPerHour.length; i++){
+  var tableData = document.createElement('td');
+  tableData.textContent = dubai.cookiesPerHour[i];
+  tableRow.appendChild(tableData);
+}  
+parentElement.appendChild(tableRow);
+
+
+// ROW FIVE: Paris
+
+var tableRow = document.createElement('tr');
+var cityParis = document.createElement('th');
+cityParis.textContent = paris.name;
+tableRow.appendChild(cityParis);
+for (var i = 0; i < paris.cookiesPerHour.length; i++){
+  var tableData = document.createElement('td');
+  tableData.textContent = paris.cookiesPerHour[i];
+  tableRow.appendChild(tableData);
+}  
+parentElement.appendChild(tableRow);
+
+
+// ROW SIX: Lima
+
+var tableRow = document.createElement('tr');
+var cityLima = document.createElement('th');
+cityLima.textContent = lima.name;
+tableRow.appendChild(cityLima);
+for (var i = 0; i < lima.cookiesPerHour.length; i++){
+  var tableData = document.createElement('td');
+  tableData.textContent = lima.cookiesPerHour[i];
+  tableRow.appendChild(tableData);
+}  
+parentElement.appendChild(tableRow);
+
+
+// ROW SEVEN: Totals
+
+var tableRow = document.createElement('tr');
+var rowTotals = document.createElement('th');
+rowTotals.textContent = ('Totals');
+tableRow.appendChild(rowTotals);
+for (var i = 0; i < hoursOfOperation.length; i++){
+  var tableData = document.createElement('td');
+  tableData.textContent = hoursOfOperation[i];
+  tableRow.appendChild(tableData);
+}  
+parentElement.appendChild(tableRow);
```

Comment:
This isn't calculating the totals for each store it is just returning the times again